drivers/espi/espi_mchp_xec_v2.c

```diff
@@ -1561,3 +1561,282 @@ static int espi_xec_init(const struct device *dev)
 
 	return ret;
 }
+
+#ifdef CONFIG_FLASH_HAS_DRIVER_ENABLED
+#undef DT_DRV_COMPAT
+#define DT_DRV_COMPAT microchip_xec_espi_maf_flash
+
+#include <zephyr/drivers/flash.h>
+
+#define MAX_FLASH_REQUEST 64U
```

Comment:
Add comment indicate this is maximum defined by eSPI spec

Reply:
it's not the eSPI spec max.  eSPI flash channel spec is 64,128,256,512,1024,2048,4096.  I can probably change the code to read the flash access channel configuration register to determine the configured size.  This wasn't available to me when i wrote this code, but i knew 64 was the minimum possible and should work.

Reply:
correct, my bad. I meant to say 64 it's the current max value supported by host.
Consider value used from eSPI controller register that contains the selected max payload size during eSPI handshake.
![image](https://github.com/zephyrproject-rtos/zephyr/assets/16172078/ce36fde8-9e17-4dcb-b7e6-71c0400b06c3)

Going with bare minimum is fine too.

Reply:
yes, using the exchanged size is what i plan on doing.  got busy with other things so an update is still in the works.